skymap_scanner/utils/extract_json_message.py

```diff
@@ -54,6 +54,7 @@ def extract_json_message(
     is_real_event: bool,
     cache_dir="./cache/",
     GCD_dir=None,
+    pulsesName = cfg.INPUT_PULSES_NAME
```

Comment:
Do we need to make any of these optional? Can we just set the "standard" values within the function?

Reply:
No, indeed, better to make everything explicit in the call by `start_scan()`.